apps/backend/src/build/strategy/strategies/ci/index.ts

```diff
@@ -111,29 +163,42 @@ async function getBaseScreenshotBucket(build: Build) {
     ctx,
     sha: mergeBaseCommitSha,
   });
-  return getBucketFromCommits({ shas: shas.slice(1), build: richBuild });
+
+  const baseScreenshotBucket = await getBucketFromCommits({
+    shas: shas.slice(1),
+    build: richBuild,
+  });
+
+  return {
+    baseScreenshotBucket,
+    baseBranch,
+    baseBranchResolvedFrom,
+  };
 }
 
 type CIStrategyContext = {
-  projectReferenceBranch: string;
+  checkIsReferenceBranch: (branch: string) => boolean;
```

Comment:
```suggestion
  isReferenceBranch: (branch: string) => boolean;
```

Reply:
My personal convention is: `checkIsX` for `() => boolean` `isX` for `boolean`.